autoscheduler/scraper/banner_requests.py

```diff
@@ -0,0 +1,189 @@
+""" The Banner Requests class, which is what we'll use to retrieve all of the data/courses
+    from Banner(aka Compassxe). While department scraping is pretty straightforward,
+    course scraping will need to use be concurrent/use async+await in order to minimize
+    the runtime of it.
+
+    Course retrieval example:
+        var banner = BannerRequests("201931")
+
+        depts = ["CSCE", "MATH", "CHEM"]
+        requests = [banner.search(dept) for dept in depts]
+
+        data = []
+        async with asyncio.gather(*data) as resp:
+            data.append(resp)

```

Comment:
Could we change this to 

```python
loop = asyncio.get_running_loop()
data = loop.run_until_complete(banner.search(depts))
```

This is how I have it in the old AutoScheduler repo and it runs fine